doc/news/changes/minor/20180414LucaHeltai

```diff
@@ -0,0 +1,8 @@
+New: Introduced Triangulation::reset_manifold(), as a substituted for Triangulation::set_manifold()
+with a single argument. The set_manifold() method with one argument only was used to reset the manifold
+object. This method is now deprecated, in favour of the new, more esplicative, method. 
```

Comment:
favor of the new, more explicit method

(note: American spelling of "favour")

Reply:
I don't think that you noticed the change for "esplicative". I thought you want to say "explicit", but maybe you meant "explicative".

Reply:
I don't know why it's still showing `esplicative`. Latest commit is `explicative`.

Reply:
I see the updated version now. Sorry about that :-/